packages/ui/resolution/ResolutionDecorator.js

```diff
@@ -114,6 +129,15 @@ const ResolutionDecorator = hoc(defaultConfig, (config, Wrapped) => {
 			if (config.dynamic) window.addEventListener('resize', this.handleResize);
 			// eslint-disable-next-line react/no-find-dom-node
 			this.rootNode = ReactDOM.findDOMNode(this);
+
+			updateFontScale(this.props.fontScale);
```

Comment:
```suggestion
```
I think we should move `updateFontScale` into the constructor before init function call